ext/puma_http11/extconf.rb

```diff
@@ -33,26 +33,27 @@
 
     ssl_h = "openssl/ssl.h".freeze
 
-    # below is  yes for 1.0.2 & later
+    puts "\n──── Below are yes for 1.0.2 & later ────"
     have_func "DTLS_method"                            , ssl_h
     have_func "SSL_CTX_set_session_cache_mode(NULL, 0)", ssl_h
+    have_func "SSL_CTX_set_ecdh_auto(NULL, 0)"         , ssl_h
 
-    # below are yes for 1.1.0 & later
+    puts "\n──── Below are yes for 1.1.0 & later ────"
     have_func "TLS_server_method"                      , ssl_h
     have_func "SSL_CTX_set_min_proto_version(NULL, 0)" , ssl_h
-
-    # below are yes for 1.1.1 & later
-    have_func "SSL_CTX_set_ciphersuites(NULL, \"\")"   , ssl_h
-
     have_func "X509_STORE_up_ref"
```

Comment:
Should `X509_STORE_up_ref` be under 1.1.1? It moved here

https://www.openssl.org/docs/man1.1.0/man3/X509_STORE_up_ref.html is "Page Not Found" but https://www.openssl.org/docs/man1.1.1/man3/X509_STORE_up_ref.html exist

Reply:
OpenSSL docs, especially older ones...

See https://github.com/openssl/openssl/blob/OpenSSL_1_1_0/crypto/x509/x509_lu.c#L220

Reply:
I just updated the PR, added links for the two 'special items'...